### PR TITLE
Websocket api functionality + environment vars for deployment

### PR DESCRIPTION
**Pull request by Digit4**
Implemented basic WebSocket API Functionality and a minimal Frontend to test the same.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,23 @@ A multiplayer [Chidiya Uddi](https://dsource.in/resource/indian-games/indoor-gam
 # Contribution Rules
 
 1. Simple: fork the repo, make meaningful and valuable code updates, create a pr.
-2. If you make GPT write all the code for your PRs, then _aandi mandi sandi..._ applies!
```
Comment (RTG2717) [inline README.md:8]: Please put that back, it's fine if you use GPT for some parts but let this be here anyway, for the jokes maybe!

**Comment by RTG2717** [comment]
Thanks buddy, this seems pretty cool. Just instead of mouse positions - we will have to send up/down info only (well, it can be edited later easily). Give me some time, I will run it once by tonight and then merge.

**Comment by Digit4** [comment]
environment vars is an addon, to test only use until [4e5ad44](https://github.com/RTG2717/iso-chidiya-uddi/pull/5/commits/4e5ad4478c88021af17c1fc17f6f74bd22801f33)

**Comment by RTG2717** [comment]
Cool, works great, closing!!

**Pull request merged**